README.md

```diff
@@ -94,6 +94,30 @@ Raise an exception if not successful with `run!`:
 Scmd.new("cd /path/that/does/not/exist").run! #=> Scmd::Command::Failure
 ```
 
+### Environment variables
+
+Pass environment variables:
+
+```ruby
+cmd = Scmd.new("echo $TEST_VAR", {
+  :env => 'TEST_VAR' => 'hi'
```

Comment:
@jcredding missing `{}` around the value of the `:env` key.  Or better, I'd multiline it for the README to make sure it is easy to read.